packages/config/config/devices/0x0234/zif5028_heatit_z-din_616.json

```diff
@@ -101,6 +200,18 @@
 			]
 		},
 		"4": {
+			"label": "Timer for input 1",
+			"description": "Configures the timer of input 1",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5": {
 			"label": "Function setup of input 2.",
 			"description": "Configures the function of input 2.",
```

Comment:
Unnecessary description, please remove. Same for param 3 please, I cannot select that here.